package.json

```diff
@@ -10,23 +10,27 @@
         "eslint:watch": "nodemon --watch . --ext ts,vue --exec npm run --silent eslint:check",
         "gettext:extract": "vue-gettext-extract",
         "gettext:compile": "vue-gettext-compile",
+        "prettier:check": "prettier arches_lingo/src --check",
+        "prettier:fix": "prettier arches_lingo/src --write",
         "ts:check": "vue-tsc --noEmit",
         "ts:watch": "vue-tsc --watch --noEmit",
         "start": "cross-env NODE_OPTIONS=--max-old-space-size=2048 webpack serve --config ./webpack/webpack.config.dev.js",
         "vitest": "vitest --run --coverage"
     },
     "devDependencies": {
-        "arches-dev-dependencies": "archesproject/arches-dev-dependencies#dev/7.6.x"
+        "arches-dev-dependencies": "archesproject/arches-dev-dependencies#d75232c50957237421aece034456b785ad05bb85"
```

Comment:
I believe it would be better to find a solution instead of a workaround here. I've noticed `package-lock.json` hasn't been updated -- could that be the culprit?

Reply:
Apologies - I didn't commit `package-lock.json`, but it was updated when I used the direct commit hash. When reverting back to `dev/7.6.x` I experience the same issue. It seems that npm isn't pulling the latest from the dev-deps.

I totally agree that it's suboptimal to de-standardize `package.json`...

Reply:
:facepalm: fixed by removing & recreating package-lock.json